src/app/shared/models/settings.ts

```diff
@@ -2,4 +2,6 @@ export interface Settings {
    showSettings: boolean;
    openLinkInNewTab: boolean;
    theme: string;
+   titleFontSize: string;
```

Comment:
We can't save integers to localStorage can we? Strings are fine if that's the case :+1: